deployments/references.json

```diff
@@ -3,47 +3,69 @@
     "1": "ethereum",
     "5": "ethereum-goerli-testnet",
     "10": "optimism",
+    "30": "rsk",
+    "31": "rsk-testnet",
     "56": "bsc",
     "97": "bsc-testnet",
     "100": "gnosis",
     "137": "polygon",
     "250": "fantom",
+    "338": "cronos-testnet",
     "420": "optimism-goerli-testnet",
     "1101": "polygon-zkevm",
     "1284": "moonbeam",
     "1285": "moonriver",
     "1287": "moonbeam-testnet",
     "1442": "polygon-zkevm-goerli-testnet",
+    "2221": "kava-testnet",
+    "2222": "kava",
     "4002": "fantom-testnet",
+    "5000": "mantle",
+    "5001": "mantle-goerli-testnet",
+    "8453": "base",
     "10200": "gnosis-testnet",
     "42161": "arbitrum",
     "43113": "avalanche-testnet",
     "43114": "avalanche",
+    "59140": "linea-goerli-testnet",
+    "59144": "linea",
     "80001": "polygon-testnet",
+    "84531": "base-goerli-testnet",
     "421613": "arbitrum-goerli-testnet",
     "11155111": "ethereum-sepolia-testnet"
   },
   "MerkleFunder": {
     "1": "0x04d2B3DdCdb2790571Ca01F4768e3cC98FCb0D2B",
     "5": "0x04d2B3DdCdb2790571Ca01F4768e3cC98FCb0D2B",
     "10": "0x04d2B3DdCdb2790571Ca01F4768e3cC98FCb0D2B",
+    "30": "0x04d2B3DdCdb2790571Ca01F4768e3cC98FCb0D2B",
+    "31": "0x04d2B3DdCdb2790571Ca01F4768e3cC98FCb0D2B",
     "56": "0x04d2B3DdCdb2790571Ca01F4768e3cC98FCb0D2B",
     "97": "0x04d2B3DdCdb2790571Ca01F4768e3cC98FCb0D2B",
     "100": "0x04d2B3DdCdb2790571Ca01F4768e3cC98FCb0D2B",
     "137": "0x04d2B3DdCdb2790571Ca01F4768e3cC98FCb0D2B",
     "250": "0x04d2B3DdCdb2790571Ca01F4768e3cC98FCb0D2B",
+    "338": "0x6adD2B0D2bA7A4075d75c3E1801214c53e407418",
```

Comment:
I'm guessing cronos does not support deterministic deployments, right?

Reply:
Yes :(